src/java/arjdbc/util/ActiveRecord.java

```diff
@@ -0,0 +1,36 @@
+package arjdbc.util;
+
+import org.jruby.RubyClass;
+import org.jruby.RubyModule;
+import org.jruby.runtime.ThreadContext;
+
+/**
+ * Provide a standard place to get references to ActiveRecord classes and modules
+ */
+public class ActiveRecord {
```

Comment:
I dislike this type largely because although they are static methods we continuously need to look these up in constant maps over and over.  Older code also did this but making this class I think will cement this design.  I would rather our RubyJdbc java class receive these as field in the constructor or somehow save them once they are retrieved (but they cannot be in static fields or we will create memory leaks of the whole ruby runtime0.